cfgov/regulations3k/parser/paragraphs.py

```diff
@@ -36,6 +38,32 @@ def graph_top(graph_text):
         '\xa7')[0][:200]
 
 
+def lint_paragraph(graph_text):
+    """Clean formatting anomalies.
+
+    - Missing em dashes
+    - restoring italics
+    """
+    fix1 = restore_emdash(graph_text)
+    fix2 = restore_italics(fix1)
+    return fix2
+
+
+def restore_italics(graph_text):
+    fix1 = re.sub(
+        r'\*\*(see)\*\*', '*\g<1>*', graph_text, flags=re.IGNORECASE)
+    fix2 = re.sub(
+        r'\*\*(et[.]? seq[. ]?)\*\*?', '*\g<1>*', fix1, flags=re.IGNORECASE)
```

Comment:
What're all the possible combinations for `et. seq.`? Can there be both a `.` and a space after `seq`? If so, maybe `\*\*(et\.? seq\.? ?)\*\*?` would work better? https://regexr.com/3sr2f

Reply:
Good catch! Thank you.